test/BreadcrumbItemSpec.js

```diff
@@ -36,15 +24,17 @@ describe('BreadcrumbItem', () => {
     );
 
     assert.include(ReactDOM.findDOMNode(instance).className, 'active');
+    assert.ok(ReactTestUtils.findRenderedDOMComponentWithTag(instance, 'span'));
   });
 
-  it('Should render `span` as inner element when is active', () => {
+  it('Should render `span.active` when active and has href', () => {
     const instance = ReactTestUtils.renderIntoDocument(
-      <BreadcrumbItem active>
-        Crumb
+      <BreadcrumbItem href="#" active>
```

Comment:
should probably check that `href` isn't being incorrectly flushed to the DOM here no?